src/Compilers/Core/RebuildTest/RebuildCommandLineTests.cs

```diff
@@ -167,25 +197,59 @@ End Module
         {
             var list = new List<object?[]>();
 
-            PermutateOptimizations("hw.vb /target:exe /debug:embedded", "test.exe", null);
+            Add(Permutate(new CommandInfo("hw.vb /target:exe /debug:embedded", "test.exe", null)));
 
             return list;
 
-            void PermutateOptimizations(string commandLine, string peFileName, string? pdbFileName)
+            IEnumerable<CommandInfo> Permutate(CommandInfo commandInfo)
```

Comment:
>IEnumerable [](start = 12, length = 11)

Consider adding `static` for each of these local functions.

Reply:
They can't be `static` because they manipulate `list`